defaults/main.yml

```diff
@@ -30,7 +30,7 @@ cvmfs_force_unmount: false
 # according to cvmfs_client_pv_name, cvmfs_client_vg_name, and
 # cvmfs_client_lv_name as described below. If false, you will need to
 # configure the cache storage yourself instead, and also ensure it is the required size.
-cvmfs_client_configure_storage: true
```

Comment:
This is the only aspect I changed.

Reading the description of this variable gives the impression, particularly to new users who are not familiar with the CVFS, that they need to e.g. create a new file system to hand to CVFS while instead they can simply set this to `false` and the CVFS client handles everything automatically (correct?).

Reply:
That impression is generally correct; it is recommended to create a new filesystem for the cache (/var/lib/cvmfs) so that the storage for the cache is dedicated and guaranteed to be available. If it is just in / then the CVMFS client could be deprived of cache space if any other application or process on the system uses too much space.  If `true` the ansible role sets it up for you, so isn't that the easy route? It's not related to FUSE.

Reply:
If `true`, the ansible role expects a volume group to be present, i.e. it is not automatic.
If the default behavior was fully automatic this would not be an issue.

I would just point out that the [default installation instructions for the CVMFS](https://cvmfs.readthedocs.io/en/stable/cpt-quickstart.html) correspond to the option `false` here (if I understand correctly).

Reply:
No, if true it creates a PV, VG and LV for you:
https://github.com/cvmfs-contrib/ansible-cvmfs-client/blob/master/tasks/main.yml#L12
https://github.com/cvmfs-contrib/ansible-cvmfs-client/blob/master/tasks/storage.yml
You just need to tell it the physical device on which to make them, e.g. /dev/vdb, or you can use an existing VG instead as described in the documentation.
https://github.com/cvmfs-contrib/ansible-cvmfs-client/blob/master/defaults/main.yml#L35

Reply:
The quickstart guide is just that, to start quickly and get it working, suitable for a minimal basic setup , but if you're using the Ansible role you're probably installing the client on many systems or a cluster intended for production use, and on which many other applications or users could be doing things that consume disk space, in which case the cache should be on a dedicated filesystem to ensure the right amount of space is available, corresponding to the configured value of CVMFS_QUOTA_LIMIT.

Reply:
> You just need to tell it the physical device on which to make them

What if my node/VM/... in its current setup does *have* another physical device on which to make them?

In the end, all I was trying to say is that it can be helpful if the default settings of the ansible role let it run through without extra requirements, and that from the description of this variable alone it was not clear to me that this option even exists (namely by passing `false` to `cvmfs_client_configure_storage`).

I see that you have thought about the default of this value, and so I will revert this change
(it just means that users who want to give this role a "quick try" on some VM will either need to change the value of this option or mount an extra hard disk).

Reply:
> In the end, all I was trying to say is that it can be helpful if the default variables of an ansible role let it run through without extra requirements

Yes, I agree, I can see that. I thought about it more but I considered that the risk of accidentally breaking things outweighs the convenience of just working out of the box. I'm not sure if there is an easy way to have both convenience and safety, but I recently updated the var descriptions to try to explain this better.

No worries, sorry it was not clear and sorry for the long delay @ltalirz